internal/e2e/flexible_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package e2e
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/golang-samples/internal/aeintegrate"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+func TestHelloWorld(t *testing.T) {
+	tc := testutil.EndToEndTest(t)
+
+	helloworld := &aeintegrate.App{
+		Name:      "hw",
+		Dir:       tc.Path("docs", "managed_vms", "helloworld"),
+		ProjectID: tc.ProjectID,
+	}
+
+	bodyShouldContain(t, helloworld, "/", "Hello world!")
```

Comment:
We'll definitely have tests that test different paths (e.g. for Bookshelf). These are just very basic ones.